src/mmw/apps/modeling/views.py

```diff
@@ -210,10 +213,24 @@ def start_tr55(request, format=None):
 
 
 def _initiate_tr55_job_chain(model_input, job_id):
-    return chain(tasks.make_gt_service_call_task.s(model_input),
-                 tasks.run_tr55.s(model_input),
-                 save_job_result.s(job_id, model_input)) \
-        .apply_async(link_error=save_job_error.s(job_id))
+    current_hash = model_input['modification_hash']
+    census = model_input.get('census')
+    census_hash = None
+
+    if census is not None:
+        census_hash = model_input['census'].get('modification_hash')
+
+    if census is None or current_hash != census_hash:
+        # TODO: Remove. For demo purposes only
+        time.sleep(2)
```

Comment:
I understand the temporary purpose of this sleep, though maybe it makes more sense to be in the `prepare_census` function, since that's what would get/not get called and take more/less time, depending, right?   Since it's async here, it should just return at about the same rate either way.  I realize it doesn't make a difference for testing, but I just want to make sure I'm understanding this flow.